### ISSUE TITLE
Data source requests

### ISSUE THREAD

**Issue by TimoLeung** [SaiPravallika/Skin-Disease-Recoginition#1]
Hi, what is the accuracy of your trained model on the validation set, I would like to reproduce your model training process, is it convenient to provide the dataset as well as the training code, so that I can go a step further in model training?
